### ISSUE TITLE
Cannot reproduce the results of SSD-300 on WIDER_FACE

### ISSUE THREAD

**Issue by sunke123** [open-mmlab/mmdetection#1740]
Hi @sovrasov,
Recently, I'm reproducing the results of SSD-300 on WIDER_FACE.
I use the provided config based on mmdetection and get Recall 36.9, Precision 0.023 and AP 27.3.
The results are worse than Recall 43.4, Precision 0.029 and AP 34.7 (shown in https://github.com/open-mmlab/mmdetection/pull/765#issuecomment-502579220).

Generally, the models provided by mmdetection are trained on 8 gpus. So, is the SSD-300 on WIDER_FACE also trained on 8 gpus? If so, maybe I will change the learning rate because I trained the model on 4 gpus.

Moreover, any other advice?

Thanks a lot.

**Comment by sovrasov** [comment]
The problem should be in the learning rate value.
As far as I remember this config is for one or two GPUs, so you have to increase the LR in 2 or 4 times.
That's my fault, I didn't provide the correct LR for 8 GPUs.

**Comment by sunke123** [comment]
Thanks a lot. I will try it.

**Comment by hellock** [comment]
@sovrasov Maybe you could open a PR to fix that.

**Comment by sunke123** [comment]
@sovrasov @hellock 
I have tried some training settings. In these experiments, I only changed the #GPU and LR. When adopting lr=0.0015 and training the SSD-300 on 1 GPU, I can get the comparable performance as @sovrasov.

![image](https://user-images.githubusercontent.com/11239351/70414042-e4d68480-1a93-11ea-9f14-c1a53fb6ee24.png)

**Comment by sovrasov** [comment]
Nice! According to these results most likely I had used 2 gpus with lr=1e-3.
I'll create a PR with lr changed to 8*1.5e-3 for 8 GPUs.
